ci/docker/build/Dockerfile

```diff
@@ -8,19 +8,97 @@ ENV PATH $PATH:$DEPOT_TOOLS_PATH
 # Notes:
 # - libx11-dev is used by Flutter for desktop linux (see also install-build-deps-linux-desktop.sh)
 # - chrome is used by Flutter for web.
-RUN apt-get update && \
-    apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https && \
-    git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH && \
-    mkdir --parents $ENGINE_PATH && \
-    git clone https://github.com/flutter/buildroot.git $BUILDROOT_PATH && \
-    cd $BUILDROOT_PATH && \
-    ./build/install-build-deps.sh --no-prompt && \
-    ./build/install-build-deps-android.sh --no-prompt && \
-    (echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list) && \
-    (curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -) && \
-    (wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -) && \
-    echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list && \
-    apt-get update && apt-get install -y google-cloud-sdk google-chrome-stable libx11-dev && \
+
+# Updates the distribution.
+RUN apt-get update 
```

Comment:
it's not a big deal, but the reason we did so many steps in a single `RUN` statement was to reduce the overall docker image size. each dockerfile statement is saved as its own image layer. if you download a file in one layer and then delete it in a subsequent layer, your final image will have the file size times two, whereas if you download and delete it in a single layer, the net file storage will be zero.

Reply:
I see, do you know how can I check the sizes to compare? if the increase is significant I'll get it back to multiple statements per run.

Reply:
you can pull the image you just created via: `docker pull gcr.io/flutter-cirrus/35f4dd0e84cf343e53d000bce8899df0:latest`. when I tried it locally I didn't have permissions.

Reply:
Sizes are about the same:

REPOSITORY                                                           TAG                IMAGE ID       CREATED         SIZE
gcr.io/flutter-cirrus/35f4dd0e84cf343e53d000bce8899df0               latest             76f5b861a37a   18 hours ago    2.57GB
gcr.io/flutter-cirrus/df42c0c510954e88f1278196db0afdce               latest             b9ce201299b9   4 months ago    2.38GB

0.19GB increase do you think is a good tradeoff for the readability of the file?

Reply:
SGTM